basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/RetryStep/help.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Retry the body block up to N times if its execution finishes with status ABORTED or FAILURE.
```

Comment:
A block cannot have a “status” as such. (Pending [JENKINS-26522](https://issues.jenkins-ci.org/browse/JENKINS-26522) anyway.) Like a function in a programming language, which in fact it is, it can either return a value or throw an exception. Uncaught exceptions do get mapped onto various non-successful build statuses, but you can also set build status directly, and so on, so I think this is more misleading than helpful.